docs/standard/base-types/formatting-types.md

```diff
@@ -257,16 +257,14 @@ ms.workload:
   
 <a name="stringRef"></a>   
 ### Format Strings and .NET Class Library Types  
```

Comment:
unrelated to your change but wouldn't be better to say .NET types instead of .NET class library types?